app/services/open_library_service.rb

```diff
@@ -0,0 +1,34 @@
+class OpenLibraryService
+  def initialize; end
+
+  def book_info(param)
+    response = HTTParty.get("https://openlibrary.org/api/books?bibkeys=ISBN:#{param}&format=json&jscmd=data")
```

Comment:
I would extract the base url to a secret, so we can parameterize it depending the environment on which the app is running.